src/lib/core/components/visualizations/Visualizations.scss

```diff
@@ -291,3 +291,9 @@ $data-table-inner-border: 1px solid #888;
     border-bottom: $data-table-inner-border;
   }
 }
+
+.ReactModal__Content {
```

Comment:
@chowington This might not be the best idea. If this is a global definition, it would affect all modals.

Reply:
It no longer exists :)